frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -280,6 +343,17 @@ export default function CourseDescription({ structure }) {
                   </Tag>
                 );
               })}
+            <Title level={3} className="text cs-final-attr">
+              Capacity
+            </Title>
+            {courseCapacity.enrolments ? (
+                <ProgressBar
+                  progress={Math.round((courseCapacity.enrolments/courseCapacity.capacity) * 100)}
```

Comment:
this is nice - but we should also have text to tell them the exact capacity as well.